### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by bolverk**

### CODE REVIEW

File: source/tessellation/RoundGrid.cpp [hunk 1]
```diff
@@ -1,68 +1,54 @@
...
+      d/eta_/R<0.95 ? Vector2D(0,0) : chi_*0.5*(s-r);
+    return tess.GetMeshPoint(i) + dw;
+  }
+
+  vector<Vector2D> genNewPoints(Tessellation& tess)
+  {
+    const int N = tess.GetPointNo();
+    vector<Vector2D> res(static_cast<size_t>(N));
+    for(int i=0;i<N;++i)
+      res.at(static_cast<size_t>(i)) = genNewPoint(tess,i);
```
Comment (eladtan) [inline source/tessellation/RoundGrid.cpp]: the at is much slower than []

File: source/tessellation/RoundGrid.cpp [hunk 2]
```diff
@@ -1,68 +1,54 @@
...
 			   const OuterBoundary& bc,int NumberIt,
-			   #ifdef RICH_MPI
+#ifdef RICH_MPI
 			   Tessellation const* tproc,
-			   #endif
-	Tessellation *tess)
+#endif
+			   Tessellation *tess)
 {
-	vector<int> indeces = HilbertOrder(points, static_cast<int>(points.size()));
```
Comment (eladtan) [inline source/tessellation/RoundGrid.cpp:11]: why did you remove the hilbertorder?

**Review by eladtan — CHANGES_REQUESTED** [review]

File: source/tessellation/RoundGrid.cpp (hunk 2 above)
Comment (bolverk) [reply source/tessellation/RoundGrid.cpp:11]: Hilbert order was overwritten in the previous version

File: source/tessellation/RoundGrid.cpp (hunk 1 above)
Comment (bolverk) [reply source/tessellation/RoundGrid.cpp]: fixed

**Review by eladtan — APPROVED** [review]
